Sources/UINotificationPresentationContext.swift

```diff
@@ -73,8 +73,8 @@ public final class UINotificationPresentationContext {
         notificationView.topConstraint = notificationViewTopConstraint
         
         var constraints = [
-            notificationView.leftAnchor.constraint(equalTo: containerViewController.view.leftAnchor).usingPriority(.defaultHigh),
-            notificationView.rightAnchor.constraint(equalTo: containerViewController.view.rightAnchor).usingPriority(.defaultHigh),
+            notificationView.leftAnchor.constraint(equalTo: containerViewController.view.leftAnchor).usingPriority(.required),
+            notificationView.rightAnchor.constraint(equalTo: containerViewController.view.rightAnchor).usingPriority(.required),
```

Comment:
`required` is the default value
```suggestion
            notificationView.leftAnchor.constraint(equalTo: containerViewController.view.leftAnchor),
            notificationView.rightAnchor.constraint(equalTo: containerViewController.view.rightAnchor),
```